config.yml

```diff
@@ -1,51 +1,24 @@
 experiment_name: my_example
-n_repeats: 5
+n_repeats: 1
 scenario_params_list:
  - dataset_name:
-     - 'mnist'
-     - 'cifar10'
+     - 'titanic'
    partners_count:
```

Comment:
Hmmm, I see that you needed an updated `config.yml` for your tests, but please keep it close to its current state, maybe just adding the Titanic dataset, as it is the "main" config example of the library.
(If you play with it, just don't commit it!)

Reply:
I had in mind to reverse it to its original state before merging with `master`, maybe just adding the `titanic `dataset. 

I wanted to provide a template of this file for everyone to take a test run on this. It might not be the smartest way to do this. 
Should I just reverse it now and add it to my `.gitignore` ?